server/conf/i18n/messages

```diff
@@ -11,7 +11,21 @@
 button.logout=Logout
 label.selectLanguage=Please select your preferred language.
 button.untranslatedSubmit=Submit
+# Text on a link that gives more information about CiviForm. "CiviForm" is the app name and should
+# not be translated.
+footer.aboutCiviform=About CiviForm
 footer.supportLinkDescription=For technical support, contact
+# Text in the footer. The placeholder is a link call to action to visit a website.
+footer.lookingForMoreInformation=Looking for more information and services? {0}
+# Text in the footer telling users that this is an official website of the specified Civic entity.
+footer.officialWebsiteOf=An official website of the {0}
+# Text on a link that will scroll to the top of the page
+footer.returnToTop=Return to top
+# A message in the footer directing users to technical support. The placeholder is a link to
+# send an email to CiviForm technical support.
+footer.technicalSupport=For technical support: {0}
+# Text within a link to visit the given website
+footer.visitLink=Visit {0}
```

Comment:
Ditto

Reply:
Is `visitLink` going to be used anywhere? If not, let's delete it

Reply:
done